### PR TITLE
deleted unused endpoints

### PR DESCRIPTION
**Pull request by ElnurKhalilov1995**
qwerty

### CODE REVIEW

File: .idea/compiler.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="CompilerConfiguration">
```
Comment (VahidYaserli) [inline .idea/compiler.xml:3]: duzdu

File: .idea/compiler.xml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <annotationProcessing>
+      <profile name="Annotation profile for example-spring-project" enabled="true">
+        <sourceOutputDir name="target/generated-sources/annotations" />
+        <sourceTestOutputDir name="target/generated-test-sources/test-annotations" />
+        <outputRelativeToContentRoot value="true" />
+        <processorPath useClasspath="false">
+          <entry name="$MAVEN_REPOSITORY$/org/mapstruct/mapstruct-processor/1.3.1.Final/mapstruct-processor-1.3.1.Final.jar" />
```
Comment (VahidYaserli) [inline .idea/compiler.xml:10]: salam

File: .idea/compiler.xml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+          <entry name="$MAVEN_REPOSITORY$/com/jolira/hickory/1.0.0/hickory-1.0.0.jar" />
+          <entry name="$MAVEN_REPOSITORY$/org/mapstruct/mapstruct/1.3.1.Final/mapstruct-1.3.1.Final.jar" />
+          <entry name="$MAVEN_REPOSITORY$/junit/junit/4.12/junit-4.12.jar" />
+          <entry name="$MAVEN_REPOSITORY$/org/assertj/assertj-core/3.11.1/assertj-core-3.11.1.jar" />
+          <entry name="$MAVEN_REPOSITORY$/com/google/guava/guava/19.0/guava-19.0.jar" />
+          <entry name="$MAVEN_REPOSITORY$/com/puppycrawl/tools/checkstyle/8.14/checkstyle-8.14.jar" />
+          <entry name="$MAVEN_REPOSITORY$/javax/inject/javax.inject/1/javax.inject-1.jar" />
```
Comment (VahidYaserli) [inline .idea/compiler.xml:17]: test

File: .idea/compiler.xml (hunk 1 above)
Comment (ElnurKhalilov1995) [reply .idea/compiler.xml:3]: > duzdu

mence yox

**Review by VahidYaserli — APPROVED** [review]

**Pull request merged**